scalopus_tracing/test/test_ctfevent.cpp

```diff
@@ -51,5 +52,17 @@ int main(int /* argc */, char** /* argv */)
   test(event.domain(), "scalopus_scope_id");
   test(event.name(), "exit");
   test(event.eventData().at("id"), 4144779573U);
+}
+
+void test_unexpected()
+{
+  std::string line{ "[scalopus] Cleaning up transport to: 404305\n" };
+  scalopus::CTFEvent event{ line };  // should not throw.
```

Comment:
I wonder why this test isn't a `gtest`? Then you could use `EXPECT_NO_THROW` here.

I'm just curious. I guess there's a good reason for that.

Reply:
Sounds good.